client/js/components/faq/DpFaqSupport.vue

```diff
@@ -89,10 +92,20 @@ export default {
   name: 'DpFaqSupport',
   components: { DpFaqSupportCard },
 
+  data() {
+    return {
+      contactList: this.contacts
+    }
+  },
+
   computed: {
     ...mapState('customerContact', {
       contacts: 'items'
-    })
+    }),
+
+    contactLength () {
```

Comment:
Minor nitpick: i'd prefer `contactCount` over `contactLength` because length could also be associated with the length of a string.

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/2139/commits/3d072cc4e55010337160f13e98fbe86dab0a04cd